app.js

```diff
@@ -0,0 +1,47 @@
+var express = require('express');
+var path = require('path');
+var favicon = require('serve-favicon');
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+
+var index = require('./routes/index');
+const users = require('./routes/users');
+
+const app = express();
+
+// uncomment after placing your favicon in /public
+//app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cookieParser());
+
+app.use('/', index);
+app.use('/users', users);
+
+
+// catch 404 and forward to error handler
+
+app.use(function(req, res, next) {
+    const err = new Error('Not Found');
+    err.status = 404;
+    next(err);
+});
+
+// error handler
+app.use(function(err, req, res, next) {
+    // set locals, only providing error in development
+    res.locals.message = err.message;
+    res.locals.error = req.app.get('env') === 'development' ? err : {};
+
+    // render the error page
+    res.status(err.status || 500);
+    res.render('error');
+});
+
+app.listen(3000, function() {
+    console.log('App API listening in on port 3000')
+});
```

Comment:
These 3 lines (43-45) are interfering with the app.listen that is in /bin/www which is where the project is actually run. This is apparent when you run npm start to try and test the program. You should be able to just delete these lines with no other issue.